base/xx-cc

```diff
@@ -8,15 +8,36 @@ if [ -n "$XX_MIRROR" ]; then
   mirrors="$XX_MIRROR"
 fi
 
+_wget() {
+  set +e
+  wget "$@"
+  _ret=$?
+  if [ $_ret = 0 ]; then
```

Comment:
```
if ! wget ...
  if [ .. = "gnu" ]; then return 1; fi
  installwget
  wget "$@"
fi
```
should be enough

Reply:
I wanted to preserve the original exit code but if you don't think it's worth it I'm fine with it.

Reply:
it's ignored anyway on the caller side